upstairs/src/upstairs.rs

```diff
@@ -449,12 +441,13 @@ impl Upstairs {
                 const LEAK_MS: usize = 1000;
                 let leak_tick =
                     tokio::time::Duration::from_millis(LEAK_MS as u64);
-                if let Some(iop_limit) = self.guest.get_iop_limit() {
-                    let tokens = iop_limit / (1000 / LEAK_MS);
+                let limits = *self.guest.limits.lock().unwrap();
```

Comment:
Sure, done in 84669db1de07f2a27ec8797f80bc5f22546011c2

It's starting to feel a _little_ silly: right now, they only exist for the sake of unit tests in `crucible-upstairs`, since no one is setting them to anything other than `None` in production code.  Still, further refactoring / cleanup is probably out of scope for this PR.